fe/fe-core/src/main/java/com/starrocks/common/DataQualityException.java

```diff
@@ -21,7 +21,7 @@
  * Exception for data quality check failed
  */
 public class DataQualityException extends LoadException {
-    public static final String QUALITY_FAIL_MSG = "quality not good enough to cancel";
+    public static final String QUALITY_FAIL_MSG = "quality issues with ingested data, please check trackingSQL for details";
```

Comment:
```error on ingested data```?

Reply:
`All the loading methods provided by StarRocks can guarantee atomicity. Atomicity means that the qualified data within a load job must be all successfully loaded or none of the qualified data is successfully loaded. It never happens that some of the qualified data is loaded while the other data is not. Note that the qualified data does not include the data that is filtered out due to quality issues such as data type conversion errors.`

refer from our doc: https://docs.starrocks.io/docs/loading/Loading_intro/